charbot/events.py

```diff
@@ -428,39 +428,41 @@ async def on_message(self, message: discord.Message) -> None:
             except discord.DiscordException:  # pragma: no cover
```

Comment:
can you explain some more?

Reply:
> Try-finally without and except group is valid python syntax

but we need except for handling the errors we are getting, right?

Reply:
> > Try-finally without and except group is valid python syntax
> 
> but we need except for handling the errors we are getting, right?

Here we're just making sure handling continues in case the message has already been deleted by another bot or a human moderator. Otherwsie an error would be raised and the log message will not be sent, which is suboptimal.

Currently we don't care about any exception possibly raised by the call [according to the docs](https://discordpy.readthedocs.io/en/latest/api.html#discord.Message.delete):
- Forbidden: That won't happen, if it does something has gone wrong - If you thinkg we care about that, add specific handling
- NotFound - The message already has been deleted, there's nothign we care about to do with that
- HTTPException - All we can do is log that, which is useless becasue its probably a discord error, not an error in our code

Does that make sense?

Reply:
so basically, we can just not code for the forbidden, notfound and httpexception exceptions and instead create a single try-finally block with no nested try-excepts?

Reply:
I've created some more changes, pls review